clients/da-vinci-client/src/main/java/com/linkedin/davinci/client/NonLocalAccessPolicy.java

```diff
@@ -1,21 +0,0 @@
-package com.linkedin.davinci.client;
-
-public enum NonLocalAccessPolicy {
```

Comment:
Let's leave this enum as is and mark it deprecated instead. Once we've gotten everyone to migrate off, we can remove it. Otherwise, it becomes a breaking change.

Reply:
We will bump the major version to force customer to get rid of the usage.